virtdbg/drivers/serial.c

```diff
@@ -0,0 +1,26 @@
+#include <drivers/serial.h>
+#include <arch/io.h>
+#include <stdint.h>
+
+#define SERIAL_BASE 0x3F8
+#define LSR         (SERIAL_BASE + 0x05)
+#define TXRDY       0x20
+
+void serial_init() {
+    io_write_8(0x3F8 + 3, 3); // Configure Line Control
```

Comment:
You should probably use the SERIAL_BASE macro in these ;) 
Also comment the configuration in a comment for reference